index.html

```diff
@@ -191,6 +190,30 @@ <h2 class="card-title">Our Changing Planet</h2>
           </div>
         </div>
       </div>
+      <h2>Inputs</h2>
```

Comment:
This can be wrapped into a `<section>`. As It represents a new section for `Inputs`.

Reply:
fixed, thanks